cornflow-dags/DAG/activate_dags.py

```diff
@@ -28,7 +28,8 @@ def solve(**kwargs):
         **kwargs
     )
     with dag:
-        if not app.notify:
+        notify = getattr(app, "notify", True)
+        if not notify:
```

Comment:
This change @AlejandraGalan done over your code is to make sure that this change is compatible with older versions `cornflow-client` just in case